aries-site/src/examples/components/spinner/AnnounceSpinnerExample.js

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from 'react';
+import { Box, Button, Paragraph, Spinner } from 'grommet';
+
+export const AnnounceSpinnerExample = () => {
```

Comment:
![image](https://user-images.githubusercontent.com/6320236/111539007-03193e80-8733-11eb-938e-ade2430a4449.png)

Code is showing 404

Reply:
The 404 is because this code isn't on Github yet, but once this merges it will work.

Reply:
👆🏻 yes